packages/ozone/src/db/migrations/20240506T225055595Z-message-subject.ts

```diff
@@ -0,0 +1,22 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .alterTable('moderation_event')
+    .addColumn('subjectMessageId', 'varchar')
+    .execute()
+  // support lookup for chat.bsky.moderation.getMessageContext
+  await db.schema
+    .createIndex('moderation_event_message_id_index')
+    .on('moderation_event')
+    .column('subjectMessageId')
+    .execute()
+}
+
+export async function down(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .alterTable('moderation_event')
+    .dropColumn('subjectMessageId')
+    .execute()
+  await db.schema.dropIndex('moderation_event_message_id_index').execute()
```

Comment:
I think dropping the column causes the index to be dropped too.

Reply:
sweet yup 👍